pkg/gardenadm/cmd/init/init.go

```diff
@@ -161,28 +161,28 @@ func run(ctx context.Context, opts *Options) error {
 		deployNetworkPolicies = g.Add(flow.Task{
 			Name:         "Deploying network policies",
 			Fn:           b.ApplyNetworkPolicies,
-			Dependencies: flow.NewTaskIDs(deployGardenerResourceManager, deployExtensionControllers),
+			Dependencies: flow.NewTaskIDs(waitUntilGardenerResourceManagerReady, deployExtensionControllers),
```

Comment:
A ready `gardener-resource-manager` deployment does not imply a ready `CustomResourceDefinition`, or? Should we perhaps add a separate step that directly checks the CRD?